server/accounts.go

```diff
@@ -895,33 +931,28 @@ func (a *Account) AddServiceExportWithResponse(subject string, respType ServiceR
 		a.exports.services = make(map[string]*serviceExport)
 	}
 
-	se := a.exports.services[subject]
+	ea := a.exports.services[subject]
```

Comment:
Not sure why you felt the need to change `se` for service export to `ea`?

Reply:
Tbh, I'm not sure as this was done a long time ago. altered it back.

Reply:
`ea` is for `exportAuth` I assume, but see my other comment. This is still a serviceExport...